src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.tsx

```diff
@@ -88,13 +96,19 @@ type Gene = Pick<
 > &
   Pick3<FullGene, 'slice', 'strand', 'code'> &
   Pick3<FullGene, 'slice', 'location', 'length'> & {
-    transcripts: { stable_id: string }[];
+    transcripts: Transcript[];
   };
 
+export const isTranscriptProteinCoding = (transcripts: Transcript[]) => {
+  const proteinCodingTranscripts = transcripts.filter((transcript) =>
+    isProteinCodingTranscript(transcript)
+  );
+  return proteinCodingTranscripts.length > 0;
```

Comment:
`return transcripts.some(...)`

Reply:
Also, the function is doing something different from what its name is saying. The name is about a single transcript; but the function receives multiple transcripts. `isTranscriptProteinCoding ` -> `isProteinCodingGene`?